### PR TITLE
Document (and lint) admin helper scripts

### PR DESCRIPTION
**Pull request by yvanzo**
For all helper scripts under `admin/`:

* Support `help` command and `--help` and `-h` options
* Complete usage and error messages
* Update documentation
* Lint scripts a bit using shellcheck

### CODE REVIEW

File: admin/create-amqp-extension [hunk 1]
```diff
@@ -6,16 +6,20 @@ set -e -u
...
 
-if [ $# -ne 0 ]
+if [[ $# -ne 0 && $1 =~ -*h(elp)? ]]
 then
-  echo >&2 "$SCRIPT_NAME: too many arguments"
-  echo >&2 "$HELP"
+  echo "$HELP"
+  exit
+elif [[ $# -ne 0 ]]
+then
```
Comment (SuperSandro2000) [inline admin/create-amqp-extension:20]:
```suggestion
elif [[ $# -ne 0 ]]; then
```

File: admin/create-amqp-extension [hunk 2]
```diff
@@ -6,16 +6,20 @@ set -e -u
...
 EOH
 )
 
-if [ $# -ne 0 ]
+if [[ $# -ne 0 && $1 =~ -*h(elp)? ]]
 then
-  echo >&2 "$SCRIPT_NAME: too many arguments"
-  echo >&2 "$HELP"
+  echo "$HELP"
+  exit
```
Comment (SuperSandro2000) [inline admin/create-amqp-extension]:
```suggestion
  exit 0
```

File: admin/setup-amqp-triggers [hunk 3]
```diff
@@ -94,9 +107,13 @@ case "$1" in
 
     $DOCKER_COMPOSE_CMD exec musicbrainz rm -fr "$REMOTE_TRIGGERS_DIR"
     ;;
+  help|-h|--help)
+    echo "$HELP"
+    exit
```
Comment (SuperSandro2000) [inline admin/setup-amqp-triggers]:
```suggestion
    exit 0
```

**Review by SuperSandro2000 — COMMENTED** [review]
:)

File: admin/create-amqp-extension (hunk 2 above)
Comment (yvanzo) [reply admin/create-amqp-extension]: Changed in https://github.com/metabrainz/musicbrainz-docker/pull/131/commits/edb9b3b5fd0c25fdd0cf3eb017b6455d226da57c along with comments to document each exit status.

File: admin/setup-amqp-triggers (hunk 3 above)
Comment (yvanzo) [reply admin/setup-amqp-triggers]: Changed as well, see above.

File: admin/create-amqp-extension (hunk 1 above)
Comment (yvanzo) [reply admin/create-amqp-extension:20]: Not needed, semicolons are avoided as much as possible. Removed an errant one in https://github.com/metabrainz/musicbrainz-docker/pull/131/commits/9aa4495434916e65bf136218079e8c754de83dc0.

(Just coding style anyway.)

**Comment by yvanzo** [comment]
Thanks @SuperSandro2000!

**Comment by SuperSandro2000** [comment]
No problem. I like to help where I can.

**Pull request merged**